mathesar/state/django.py

```diff
@@ -32,11 +32,10 @@ def clear_dj_cache():
 
 
 def reflect_db_objects(metadata, db_name=None):
-    sync_databases_status()
     databases = models.Database.current_objects.all()
     if db_name is not None:
         databases = databases.filter(name=db_name)
-
+    sync_databases_status(databases)
```

Comment:
Before we were reflecting the status of every database, irrespective of whether or not we're intending to reflect only a single database. Now we'll only check the status of the database we've singled out (when we've singled a database out).